.github/workflows/ci.yaml

```diff
@@ -33,15 +38,13 @@ jobs:
           activate-environment: test_env_xgcm # Defined in ci/environment*.yml
           auto-update-conda: false
           python-version: ${{ matrix.python-version }}
```

Comment:
Just trying to understand this more. Is this line here installing a specific python version? Or is that set somewhere else?